# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xfce4-screenshooter-plugin/hotfix-nodocs.patch.cross
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xfce4-screenshooter-1.11.2/src/meson.build.vanilla	2025-05-23 14:58:38.316881808 +0200
+++ xfce4-screenshooter-1.11.2/src/meson.build	2025-05-23 14:58:46.237850704 +0200
@@ -45,7 +45,7 @@
 )
 
 help2man = find_program('help2man', required: false)
-if help2man.found()
+if false
   custom_target(
     'xfce4-screenshooter.1',
     depends: screenshooter,
